packages/react-juce/src/lib/Backend.ts

```diff
@@ -144,6 +222,100 @@ export class ViewInstance {
         NativeMethods.setViewProperty(this._id, k, v);
       return;
     }
+
+    // Look for border properties and translate into our internal
+    // border-info property.
+    if (propKey.startsWith("border")) {
```

Comment:
I think we can probably extract a border property match/normalize utility out of this. We did something like that for colors originally, but after a bit of searching through RN's source I found this: https://github.com/facebook/react-native/tree/1465c8f3874cdee8c325ab4a4916fda0b3e43bdb/packages/normalize-color. I think we should probably replace our own color parser with their `normalize-color` and try to refactor the code below into our own `normalize-border` package. What do you think?

Reply:
Using `normalize-color` would certainly improve colour specification flexibility. No harm in putting border parsing into its own module.